Modules/IO/GDCM/test/itkGDCMImageIOIntToUintTest.cxx

```diff
@@ -0,0 +1,77 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkGDCMImageIO.h"
+#include "itkImageFileReader.h"
+#include "itkTestingMacros.h"
+
+// Specific ImageIO test
+
+/** This test verifies itkGDCMImageIO can read
```

Comment:
Could `itkGDCMImageReadWriteTest.cxx` have been used here instead of writing a new test? That could also be used to regress the read image against a baseline in MetaImage or NRRD format.

Reply:
Yes the demo image is a very simple image with some flat colors, I'm not surprised compression is efficient there!
![image](https://github.com/user-attachments/assets/bcc7d897-d0b0-4010-a181-73d136659e83)